### PR TITLE
DOC-2285: Removal of deprecated Export Premium Plugin for TinyMCE 7.0.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2285

Changes:
* Removal of deprecated Export Premium Plugin for `TinyMCE 7.0`. `20 files changed`

* delete `modules/ROOT/examples/live-demos/export/index.html`
* delete `modules/ROOT/examples/live-demos/export/index.js`
* delete `modules/ROOT/pages/export.adoc`
* delete `modules/ROOT/partials/commands/export-cmds.adoc`
* delete `modules/ROOT/partials/configuration/export_ignore_elements.adoc`
* delete `modules/ROOT/partials/events/export-events.adoc`
* delete `modules/ROOT/partials/menu-item-ids/export-menu-items.adoc`
* delete `modules/ROOT/partials/plugin-apis/export-apis.adoc`
* delete `modules/ROOT/partials/plugin-files/plugin-file-list-export.adoc`
* delete `modules/ROOT/partials/toolbar-button-ids/export-toolbar-buttons.adoc`

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Comment by FarzadHayat** [comment]
Do references to `image_proxy` and `image_cors_hosts` need to be removed?
image

**Review by FarzadHayat — CHANGES_REQUESTED** [review]
Do references to `image_proxy` and `image_cors_hosts` need to be removed? (require advice from devs)
image

**Review by FarzadHayat — APPROVED** [review]

File: modules/ROOT/partials/configuration/icon_list.adoc [hunk 1]
```diff
@@ -58,7 +58,6 @@
 | `+embed-page+` | image:icons/embed-page.svg[embed-page.svg] | `+embed-page.svg+`
 | `+embed+` | image:icons/embed.svg[embed.svg] | `+embed.svg+`
 | `+emoji+` | image:icons/emoji.svg[emoji.svg] | `+emoji.svg+`
-| `+export+` | image:icons/export.svg[export.svg] | `+export.svg+`
```
Comment (ltrouton) [inline modules/ROOT/partials/configuration/icon_list.adoc:61]: I am not 100% sure if the icon will be removed actually. It is still currently in the codebase

**Review by ltrouton — APPROVED** [review]

File: modules/ROOT/partials/configuration/icon_list.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/partials/configuration/icon_list.adoc:61]: Im not entirely sure, as this is the first time I have removed a plugin form the docs at this level.

Comment (kemister85) [reply modules/ROOT/partials/configuration/icon_list.adoc:61]: Removing it from here will only remove it from the icon_list in the documentation page for 7.0. "marking as resolved for now"

**Review by ShiridiGandham — COMMENTED** [review]
@kemister85 Please wait for me to confirm this removal again.. just in case. We can merge in next couple of day.

**Pull request merged**